extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking.tsx

```diff
@@ -46,65 +46,63 @@ function PanelMeasurementTableTracking({
   );
 
   return (
-    <>
-      <StudySummaryFromMetadata studyInstanceUID={trackedStudy} />
-      <PanelMeasurement
-        servicesManager={servicesManager}
-        extensionManager={extensionManager}
-        commandsManager={commandsManager}
-        measurementFilters={measurementFilters}
-        customHeader={({ additionalFindings, measurements }) => {
-          const disabled = additionalFindings.length === 0 && measurements.length === 0;
+    <PanelMeasurement
+      servicesManager={servicesManager}
+      extensionManager={extensionManager}
+      commandsManager={commandsManager}
+      measurementFilters={measurementFilters}
+      title="Measurements"
+      customHeader={({ additionalFindings, measurements }) => {
+        const disabled = additionalFindings.length === 0 && measurements.length === 0;
 
-          if (disableEditing || disabled) {
-            return null;
-          }
+        if (disableEditing || disabled) {
+          return null;
+        }
 
-          return (
-            <div className="bg-background flex h-9 w-full items-center rounded pr-0.5">
-              <div className="flex space-x-1">
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-1.5"
-                  onClick={() => {
-                    commandsManager.runCommand('clearMeasurements', measurementFilters);
-                  }}
-                >
-                  <Icons.Download className="h-5 w-5" />
-                  <span className="pl-1">CSV</span>
-                </Button>
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-0.5"
-                  onClick={() => {
-                    sendTrackedMeasurementsEvent('SAVE_REPORT', {
-                      viewportId: viewportGrid.activeViewportId,
-                      isBackupSave: true,
-                    });
-                  }}
-                >
-                  <Icons.Add />
-                  Create SR
-                </Button>
-                <Button
-                  size="sm"
-                  variant="ghost"
-                  className="pl-0.5"
-                  onClick={() => {
-                    commandsManager.runCommand('clearMeasurements', measurementFilters);
-                  }}
-                >
-                  <Icons.Delete />
-                  Delete All
-                </Button>
-              </div>
+        return (
+          <div className="bg-background flex h-9 w-full items-center rounded pr-0.5">
+            <div className="flex space-x-1">
+              <Button
+                size="sm"
+                variant="ghost"
+                className="pl-1.5"
+                onClick={() => {
+                  commandsManager.runCommand('clearMeasurements', measurementFilters);
```

Comment:
Ouch - this one is running clear measurements instead of saveCSV - can you fix that so it runs saveCSV (or whatever the command name is to save the CSV)
Also, can you pass in the nested measurement filter instead of the top level measurement filter for all 3 commands - that is the new filter I (just) asked to be created in the PanelMeasurement table - that way the command applies to the correct set of data.